.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": "always"
```

Comment:
Because we have 'import/organize-imports' rule in eslint it would be nice to auto organize them in vscode